source/basic.tex

```diff
@@ -5487,7 +5487,7 @@
 \pnum
 If the initialization of a non-local variable with static or thread storage duration
 exits via
-an exception, \tcode{std::terminate} is called\iref{except.terminate}.%
+an exception, \tcode{std::terminate()} is called\iref{except.terminate}.%
```

Comment:
We have a mix of "`std::terminate()` is called" and "`std::terminate` is called" here, with some examples changing in one direction and some changing in the other direction. I think we have guidance that "the function `std::terminate` is called" should hot have parentheses, but these cases without a "the function" prefix are ambiguous between calling a function and "calling" (evaluating) a function call expression.

Reply:
Ok, going for "the function std::terminate is called" globally.